### PR TITLE
Small Improvements

### PR DESCRIPTION
**Pull request by nalourie-ai2**
This pull request delivers a few small improvements to the library that I found while reading through some of it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,10 +78,17 @@ Create a Python 3.6 virtual environment, and install the necessary requirements
...
   ```
 
-Changing the flag to false if you don't want to be able to run tests.
+Changing the flag to false if you don't want to be able to run
+tests. Once the requirements have been installed, run:
 
-Note that if you use the source installation, you won't be able to use the `allennlp`
-command but rather you'll need to run `./bin/allennlp`.
+  ```bash
+  pip install --editable .
```
Comment (DeNeutoy) [inline README.md]: Oh sweet! I didn't know you can do this, so useful.

File: .gitignore [hunk 2]
```diff
@@ -1,14 +1,38 @@
...
+.mypy_cache
+allennlp.egg-info/
 build/
 dist/
-.mypy_cache
+
+
+# dev tools
+
+.envrc
```
Comment (DeNeutoy) [inline .gitignore]: Put these in your global .gitignore?

**Comment by schmmd** [comment]
Hi Nelson, I think I misunderstood you in slack earlier, or at least didn't
fully understand the -e option.

File: README.md [hunk 3]
```diff
@@ -78,10 +78,17 @@ Create a Python 3.6 virtual environment, and install the necessary requirements
...
+  ```
+
+To install the AllenNLP library in `editable` mode into your
```
[the remark this answers is not in the document]
Comment (nalourie-ai2) [reply README.md]: The sentence looks good to me :)

**Comment by nalourie-ai2** [comment]
@schmmd, so am I good to merge this?

**Comment by schmmd** [comment]
@nalourie-ai2 back from a week OOO.  I'll merge this once tests pass.

**Pull request merged**